Makefile

```diff
@@ -8,40 +6,38 @@ else
 endif
 
 TARGETS=awesome
-BUILDLN=build
+BUILDDIR=build
 
-all: $(TARGETS) $(BUILDLN) ;
+all: $(TARGETS) ;
```

Comment:
Hm... but you are right, there is a problem: If you just change the source code, then I guess that `make` won't be run in `build/`. Or does the wildcard pattern match anyway?